spec/index.html

```diff
@@ -559,27 +556,28 @@ <h3>Triples</h3>
       |p| is called the <dfn>predicate</dfn> of the triple, and
       |o| is called the <dfn>object</dfn> of the triple.</p>
 
-    <p>A <dfn data-local-lt="quoted">quoted triple</dfn> is an <a>RDF triple</a> that is
-      used as the <a>subject</a> or <a>object</a> of another triple.</p>
+    <p>A <dfn data-local-lt="quoted">quoted graph</dfn> is an <a>RDF graph</a> that is
+      used as the <a>subject</a> or <a>object</a> of triple.</p>
 
     <p class="note">By the given definitions,
       the <a>subject</a> of an <a>RDF triple</a> may be either an <a>IRI</a>,
-      a <a>blank node</a>, or a <a>quoted triple</a>;
+      a <a>blank node</a>, or a <a>quoted graph</a>;
       the <a>predicate</a> of an <a>RDF triple</a> may only be an <a>IRI</a>;
       the <a>object</a> of an <a>RDF triple</a> may be either an <a>IRI</a>,
-      a <a>blank node</a>, a <a>literal</a> or a <a>quoted triple</a>.</p>
+      a <a>blank node</a>, a <a>literal</a> or a <a>quoted graph</a>.</p>
 
-    <p class="note">The definition of <a>quoted triple</a> is recursive.
-      That is, a <a>quoted triple</a> can itself have a
-      <a>subject</a> or <a>object</a> component which is another <a>quoted triple</a>.
-      However, by this definition, cycles of <a>quoted triples</a> cannot be created.</p>
+    <p class="note">The definition of <a>quoted graph</a> is recursive.
```

Comment:
Assuming you accept [my proposed change to the definition of the notion of an RDF triple](https://github.com/w3c/rdf-concepts/pull/67#issuecomment-1762820612), it is indeed the case that the definition of quoted graphs (graph terms) is recursive. In fact, the definition of the notion of an RDF graph becomes recursive. Notice, however, that the recursive nature of this definition is an implicit effect of the (new) way the notion of an RDF triple would be defined. More specifically, it is the combination of the two definitions---the definition of an RDF graph as a set of RDF triples and the definition of an RDF triple that may contain RDF graphs---that is recursive. Perhaps it is useful to call this out explicitly.

Nonetheless, as @afs already mentions somewhere else, by the way these two notions are defined, cycles are not possible.